src/Validation/Validator.php

```diff
@@ -2095,6 +2407,7 @@ public function __debugInfo()
         return [
             '_presenceMessages' => $this->_presenceMessages,
             '_allowEmptyMessages' => $this->_allowEmptyMessages,
+            '_allowEmptyFlags' => $this->_allowEmptyFlags,
```

Comment:
Should this be simplified for human use? Instead of `3` we could output `EMPTY_STRING | EMPTY_ARRAY`.

Reply:
I am not sure whether or not we should do it. If you think we should do it, I am willing to implement it.

Reply:
Maybe it isn't that important. We can move forward with this and improve it later if necessary.

Reply:
OK.